src/plugins/sync-intl/host.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = function hostSyncIntl(host) {
+	host.onRequest('intl', function() {
+		var htmlElems = document.getElementsByTagName('html');
+		if (htmlElems.length === 1 && htmlElems[0].hasAttribute('data-intl-overrides')) {
+			try {
+				return JSON.parse(
+					htmlElems[0].getAttribute('data-intl-overrides')
+				);
+			} catch (e) {}
```

Comment:
This where your _no empty catch_ override came from? :P

I usually just put an `/**/` in the block.

Reply:
It is, yeah.